desktop/apps/auction2/components/DOM.js

```diff
@@ -1,18 +1,22 @@
 import ConfirmRegistrationModal from 'desktop/components/credit_card/client/confirm_registration.coffee'
 import PropTypes from 'prop-types'
-import { Component } from 'react'
 import infiniteScroll from 'desktop/apps/auction2/utils/infiniteScroll'
 import mediator from 'desktop/lib/mediator.coffee'
 import scrollToTop from 'desktop/apps/auction2/utils/scrollToTop'
+import { Component } from 'react'
+import { connect } from 'react-redux'
 
-export default class DOM extends Component {
+class DOM extends Component {
   static propTypes = {
     auction: PropTypes.object.isRequired,
     children: PropTypes.node.isRequired,
-    store: PropTypes.object.isRequired,
     user: PropTypes.object
   }
 
+  static contextTypes = {
```

Comment:
It looks like this only needs to access the `context` so that it can pass the `store` to `infiniteScroll`, which only appears to need `dispatch`, to dispatch a single action. Haven't thought this through in detail, but could you use `mapPropsToDispatch` to get a much more specific dependency to pass on, and avoid the need to use `context`? You know I love me some `context`, but I do think it should only be used systematically.